Framework/DataHandling/inc/MantidDataHandling/GenerateGroupingPowder.h

```diff
@@ -36,6 +39,9 @@ class MANTID_DATAHANDLING_DLL GenerateGroupingPowder final : public API::Algorit
 
   void init() override;
   void exec() override;
+
+  GroupingWorkspace_sptr groupWS;
```

Comment:
This variable should be named `m_groupWS` as per the [mantid naming conventions](https://developer.mantidproject.org/Standards/CPPStandards.html#naming-conventions)